angular/projects/researchdatabox/form/src/app/component/textarea.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, Input } from '@angular/core';
+import { FormFieldBaseComponent, FormFieldCompMapEntry, FormFieldModel } from "@researchdatabox/portal-ng-common";
+import { TextareaComponentConfig } from '@researchdatabox/sails-ng-common/dist/src/config/component/textarea.model';
+import { get as _get, isUndefined as _isUndefined, isEmpty as _isEmpty } from 'lodash-es';
+
+export class TextareaModel extends FormFieldModel<string> {
+}
+
+@Component({
+  selector: 'redbox-textarea',
+  template: `
+    @if (isVisible) {
+      <ng-container *ngTemplateOutlet="getTemplateRef('before')" />
+      <textarea [formControl]="formControl"
+        class="form-control"
+        [class.is-valid]="isValid"
+        [class.is-invalid]="!isValid"
+        [attr.required]="isRequired === true ? true : null"
+        [attr.disabled]="isDisabled ? 'true' : null"
+        [attr.readonly]="isReadonly ? 'true' : null"
+        [attr.title]="tooltip ? tooltip : tooltipPlaceholder"
+        [attr.rows]="rows"
+        [attr.cols]="cols"
```

Comment:
Should be able to do the same for the rows and cols

```suggestion
       [rows]="rows"
       [cols]="cols"
```